### PR TITLE
How-To: Monitor custom applications

### PR DESCRIPTION
**Pull request by glrf**
## Checklist

- [x] Try to isolate changes into separate PRs (to build a better changelog).
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `how-to`, `tutorial`, `reference`, `explanation`, `cicd`, `dependency`
      as they show up in the changelog

### CODE REVIEW

File: docs/modules/ROOT/pages/how-to/monitor-application.adoc [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+    rules:
+    - alert: TooManyrequests
+      expr: rate(http_requests_total[1m]) > 1 <1>
+      labels:
+        severity: info
+      annotations:
+        summary: "There are a lot of requests to the application"
+YAML
+----
+<1> The alert rule will fire if the application receives more than 60 request in one minute.
```
Comment (simu) [inline docs/modules/ROOT/pages/how-to/monitor-application.adoc:188]:
```suggestion
<1> The alert rule will fire if the application receives more than 60 requests in one minute.
```

File: docs/modules/ROOT/pages/how-to/monitor-application.adoc [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+Navigate to the OpenShift Web Console https://console.<CLUSTER>.cloud/dev-monitoring/ns/<APP_NAMESPACE>/alerts.
++
+image::monitoring/custom-metrics-alert-list.png[List of alerts https://console.<CLUSTER>.cloud/dev-monitoring/ns/<APP_NAMESPACE>/alerts]
++
+The alert should be displayed in the list of alerts.
++
+[TIP]
+====
+It may take a few minutes for the alert to show up.
+If it doesn't show up that most likely means that the `PrometheusRule` CRD isn't well-formed and might have failed to apply, or that the PromQL expression isn't valid.
```
Comment (simu) [inline docs/modules/ROOT/pages/how-to/monitor-application.adoc:203]:
```suggestion
If it doesn't show up, that most likely means that the `PrometheusRule` CRD isn't well-formed and might have failed to apply, or that the PromQL expression isn't valid.
```

File: docs/modules/ROOT/pages/how-to/monitor-application.adoc [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+done
+----
+Navigate to the OpenShift Web Console https://console.<CLUSTER>.cloud/dev-monitoring/ns/<APP_NAMESPACE>/alerts.
++
+The alert should now be firing.
++
+image::monitoring/custom-metrics-alert-fire.png[Firing alert https://console.<CLUSTER>.cloud/dev-monitoring/ns/<APP_NAMESPACE>/alerts]
++
+TIP: You can send the alert to a receiver such as email, PagerDuty, or Opsgenie.
+//TODO Fix link
```
Comment (simu) [inline docs/modules/ROOT/pages/how-to/monitor-application.adoc:231]: side-note: How will we keep track of this?

**Review by simu — APPROVED** [review]
LGTM, I didn't test the instructions.

File: docs/modules/ROOT/pages/how-to/monitor-application.adoc (hunk 3 above)
Comment (glrf) [reply docs/modules/ROOT/pages/how-to/monitor-application.adoc:231]: I'll add comment to double check this link in https://github.com/appuio/appuio-cloud-docs/pull/140?

**Comment by glrf** [comment]
> Also, what's up with this link that needs to be fixed? How is it broken now, and what is holding us back to fix it now?

Because that how-to is currently being written by @bastjan, so I'm not sure yet how this link will look

**Review by corvus-ch — APPROVED** [review]

**Pull request merged**